community.main/pom.xml

```diff
@@ -23,24 +23,9 @@
         <developerConnection>${project.scm.connection}</developerConnection>
     </scm>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.kaazing</groupId>
-            <artifactId>community.license</artifactId>
-            <!-- We really want -->
-            <!--     <version>{project.version}</version> -->
-            <!-- but due to inheritance rules, this won't work. -->
-            <version>develop-SNAPSHOT</version>
```

Comment:
I was only trying to point out that the comment you removed is relevant for the code you added.  But, we can continue to discuss in that comment, as you seem to understand the problem there.